### PR TITLE
Create a workbook for microsoftdefenderforidentity

### PR DESCRIPTION
**Pull request by samikroy**
Required items, please complete
   
   Change(s):
   - Workbooks/defenderforidentity.json
   - Workbooks/WorkbooksMetadata.json
   - Workbooks/Images/Preview/microsoftdefenderforidentity-black.png
   - Workbooks/Images/Preview/microsoftdefenderforidentity-white.png

   Reason for Change(s):
   - New Workbook Addition with the below tabs
Overview
Logon Events
Query Events
Directory Events
Alerts

   Testing Completed:
   - Tested with data 

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

**Comment by samikroy** [comment]
@Azure/sentinel-repo-books-reviewers - Request your review.

File: Workbooks/WorkbooksMetadata.json [hunk 1]
```diff
@@ -1599,5 +1599,18 @@
         "templateRelativePath": "MicrosoftSentinelDeploymentandMigrationTracker.json",
         "subtitle": "",
         "provider": "Microsoft Sentinel Community"
+      },
+      {
+        "workbookKey": "MicrosoftDefenderForIdentity",
+        "logoFileName": "",
+        "description": "Use this workbook to analyse the advance hunting data ingested for Defender For Identity.",
+        "dataTypesDependencies": [],
```
Comment (v-atulyadav) [inline Workbooks/WorkbooksMetadata.json:1607]: Hi @samikroy, please add dataTypesDependecies.

Comment (samikroy) [reply Workbooks/WorkbooksMetadata.json]: > Hi @samikroy, please add dataTypesDependecies.

@v-atulyadav - Updated as requested.

**Review by v-atulyadav — APPROVED** [review]
Looks good

**Comment by samikroy** [comment]
> [v-atulyadav](/v-atulyadav)

@v-atulyadav - Could you please help with further approvals for merge.

**Review by NikTripathi — APPROVED** [review]

**Pull request merged**